build.gradle.kts

```diff
@@ -119,11 +118,9 @@ tasks.create("generateDebugRelease") {
 }
 
 tasks.create("deployToCloud") {
-    dependsOn(getTasksByName("publishToMavenLocal", true))
     dependsOn("registerCloudReleaseTag", ":cloudscrambles:appengineDeploy")
```

Comment:
Not fully related to that PR, but I'm very curious to see all these cloud-related tasks.
What's the idea behind that and the concrete use-case?
I'm not sure I'm a huge fan of having google dependencies for no apparent reasons :s
Happy to hear more about that!

Reply:
This was a proof-of-concept thing I once did in October/November to procrastinate some uni work. Right now, this is not official and the Google deps are **not** included in the JAR that we distribute to Delegates.

Reply:
Check out https://wca-scrambles-unofficial.appspot.com/ for the cloud version in action